Analysis/hacc/pat_cinema.py

```diff
@@ -2,7 +2,8 @@
 """ This executable reads a JSON file and appends the metrics file with paths to images.
 The images are plotted in this executable.
 """
-
+import sys
+sys.path.append(sys.path[0]+"/..")
```

Comment:
This means your ``${PYTHONPATH}`` is wrong. You should make that correct instead of hardcoding it in a script.

Reply:
That's the current working hacc that Jesus has. I will be working on this later to create the correct python module so that it can be submitted as a pip package.

Reply:
Pascal is looking into a more permanent fix. This is temporary for now to push the rest of the refactor.